mne/io/eeglab/eeglab.py

```diff
@@ -226,6 +233,12 @@ class RawEEGLAB(_BaseRaw):
         Names or indices of channels that should be designated
         EOG channels. If 'auto', the channel names containing
         ``EOG`` or ``EYE`` are used. Defaults to empty tuple.
+    event_id : dict
+        By default, events are read from the input file by dropping every
+        non-integer part of events containing integers, and completely
+        dropping any events without integer parts. If non-integer events
+        should be read, this should be a dict mapping from their names to
+        integers, e.g. dict(fmri_scan_onset=199, recording_start=255).
```

Comment:
I mean, this IS still a problematic solution - for example, it returns zero events for the EEGLAB *example data* (because the events are called 'rt' and 'square'). But that's IMO unavoidable because in MNE, we only allow integer codes and EEGLAB allows arbitrary strings. But there is after all a very explicit log message for this, e.g., for the testing set:

```
UserWarning: Events like the following will be dropped entirely: ['rt', 'square'], 2 in total
UserWarning: Use the `event_id` keyword to include such events manually.
```